docs/actions/bazarliste.yaml

```
@@ -8,11 +8,27 @@ actions:
   # -------------------------------------
   commons:
     title: Affichage
-    width: 30%
+    width: 33%
     properties:
       search:
         label: _t(AB_bazar_commons_search_label)
         type: checkbox
+      searchfields:
+        label: _t(AB_bazar_commons_search_fields_label)
+        type: form-field
+        advanced: true
+        showif: dynamic
+        multiple: true
+        default: bf_titre
+        showOnlyFor:
+          - bazarliste
+          - bazarcarto
+      dynamic:
+        label: Rendu dynamique (version beta)
+        showOnlyFor:
+          - bazarliste
+          - bazarcarto
+        type: checkbox
```

```suggestion
        type: checkbox
        advanced: true
```

Pour moi, cette fonctionnalité est bien trop nouvelle pour la mettre tout de suite en avant dans doryphore (mode avancé uniquement)